compiler/src/iree/compiler/Dialect/Util/Analysis/Constant/OpOracle.cpp

```diff
@@ -206,7 +205,8 @@ bool isHoistableConstExprLeaf(const ConstExprAnalysis::ConstValueInfo *info) {
   // cast or packed in some successor.
   if (auto integerType = llvm::dyn_cast<IntegerType>(
           getElementTypeOrSelf(info->constValue.getType()))) {
-    if (integerType.getWidth() % 8 != 0) {
+    // Allow i4 hoisting.
```

Comment:
AFAIK, we still don't have support through the runtime for evaluating subbyte single elements (see the comments following https://github.com/openxla/iree/pull/15682#issuecomment-1906051765).  Has this changed since that discussion? If not, then we need to support that (or maybe do some bitcasting) before flipping this switch.

Reply:
Fixed!